### ISSUE TITLE
Collapse All Menus on Brickwork Footer Except C21STATUS Rewards

### ISSUE THREAD

**Issue by c21hornak** [Century21Stores/BRICKWORK#12]
Please match the same Global Footer experience that is on the rest of the site.

**Comment by juanmriosv** [comment]
@c21hornak 
Do you want me to match the footer and collapse menus but when a user clicks on the Menu it will toggle the items?

**Comment by c21hornak** [comment]
@juanmriosv yes, that. Is that possible to look and function the same as the site footer? I found some old emails where you were saying you think you can place a script to do it.

**Comment by juanmriosv** [comment]
@c21hornak The site does not currently have that functionality. As far as I remember this was requested by Elisabeth for Mobile. I believe I can place a script for both the site and Brickwork to be able to do it.
